Doc/c-api/complex.rst

```diff
@@ -25,13 +25,17 @@ pointers.  This is consistent throughout the API.
 
    The C structure which corresponds to the value portion of a Python complex
    number object.  Most of the functions for dealing with complex number objects
-   use structures of this type as input or output values, as appropriate.  It is
-   defined as::
+   use structures of this type as input or output values, as appropriate.
 
-      typedef struct {
-         double real;
-         double imag;
-      } Py_complex;
+   .. c:member:: double real
+   .. c:member:: double imag
+
+      It is defined as::
+
+         typedef struct {
+            double real;
+            double imag;
+         } Py_complex;
```

Comment:
That will render OK, but semantically it attaches the code snippet to `imag`.
To get entries for `real` and `imag` without a description, followed by more detail for `Py_complex`, the syntax is:
```suggestion
   .. c:member:: double real
                 double imag

   The structure is defined as::

      typedef struct {
          double real;
          double imag;
      } Py_complex;
```